lib/screens/splash/splash.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+import 'package:survey_flutter/gen/assets.gen.dart';
+
+class SplashScreen extends StatefulWidget {
+  const SplashScreen({Key? key}) : super(key: key);
+
+  @override
+  State<StatefulWidget> createState() => SplashScreenState();
+}
+
+class SplashScreenState extends State<SplashScreen> {
+  bool _isLogoVisible = false;
+  bool _shouldStartAnimation = false;
+
+  @override
+  void initState() {
+    super.initState();
+    _delayedLogoVisibility();
+  }
+
+  void _delayedLogoVisibility() async {
+    await Future.delayed(const Duration(seconds: 1));
+    setState(() {
+      _isLogoVisible = true;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: LayoutBuilder(builder: (context, constraints) {
+        return Stack(
+          alignment: AlignmentDirectional.center,
+          fit: StackFit.expand,
+          children: [
+            Image.asset(
+              Assets.images.splashBackground.path,
+              fit: BoxFit.cover,
+            ),
+            _buildAnimatedLogo(constraints),
+          ],
+        );
+      }),
+    );
+  }
+
+  Widget _buildAnimatedLogo(BoxConstraints constraints) {
+    return AnimatedPositioned(
+      duration: const Duration(seconds: 1),
+      top: _shouldStartAnimation ? 153.0 : constraints.maxHeight / 2.0,
+      child: AnimatedOpacity(
+        opacity: _isLogoVisible ? 1.0 : 0.0,
+        duration: const Duration(milliseconds: 500),
+        child: Image.asset(
+          Assets.images.splashLogoWhite.path,
+        ),
+        onEnd: () {
+          setState(() {
+            _shouldStartAnimation = true;
+          });
+        },
+      ),
+    );
+  }
```

Comment:
I think the Splash screen just needs to animate the logo's opacity while keeping it at the center. It's because it's easier to coordinate and synchronize the position animation with others, like blur background and form, ... in the Login screen.

In that case, we return the inner `AnimatedOpacity` directly without wrapping in `AnimatedPositioned`.

https://github.com/nimblehq/flutter-ic-khanh-thieu/assets/17972674/60f9f909-8f3d-47fa-880f-c4db3f43960a

Reply:
Good suggestion 👍, I returned the only `AnimatedOpacity` and fix all comments at https://github.com/nimblehq/flutter-ic-khanh-thieu/pull/50/commits/a3e0891d340fcf6f2aa5ce76cbb92b2e479eb1a8 🙏